src/test/shared/sam/debugger/samDebugConfigProvider.test.ts

```diff
@@ -520,6 +530,82 @@ describe('AwsSamDebugConfigurationProvider', async () => {
             assertEqualLaunchConfigs(actual, expected, appDir)
         })
 
+        it('target=code: python 3.7', async () => {
+            const appDir = pathutil.normalize(
+                path.join(testutil.getProjectDir(), 'testFixtures/workspaceFolder/python3.7-plain-sam-app/')
+            )
+            const folder = testutil.getWorkspaceFolder(appDir)
+            const c = {
+                type: AWS_SAM_DEBUG_TYPE,
+                name: 'Test debugconfig',
+                request: DIRECT_INVOKE_TYPE,
+                invokeTarget: {
+                    target: CODE_TARGET_TYPE,
+                    lambdaHandler: 'app.lambda_handler',
+                    projectRoot: 'hello_world',
+                },
+                lambda: {
+                    runtime: 'python3.7',
+                },
+            }
+            ;(c as any).__noInvoke = true
```

Comment:
What's the point of this?

Reply:
It is temporary, but there is a note here https://github.com/aws/aws-toolkit-vscode/blob/7f77a292f7ddefba28a3cd2c67c4f2cf0ec23337/src/shared/sam/debugger/awsSamDebugger.ts#L117